core/inputs/input.ts

```diff
@@ -310,13 +310,17 @@ export namespace Input {
   /**
    * Enum for alignment of inputs.
    *
+   * @deprecated Use Blockly.inputs.Align. To be removed in v11.
    */
+  /** @suppress {deprecated} */
```

Comment:
Where is the new one? It looks like `Blockly.inputs.Align` currently would point to this enum here. That's going to make the reference docs confusing, if the new path you're telling them to use says it's deprecated.

Maybe we need a new `align.ts` file that exports the enum so that this one can truly be deprecated?